### DISCUSSION TITLE
delist Binance from monero Bisq Price Index constituents

### DISCUSSION THREAD

**Discussion by chaserene** [bisq-network/bisq#7030] [Ideas]
Binance's XMRBTC pair [currently holds a 25% weight](https://bisq.wiki/Bisq_Price_Indices#Altcoins:) in the monero Bisq Price Index.

today Binance [announced](https://www.binance.com/en/support/announcement/binance-will-delist-ant-multi-vai-xmr-on-2024-02-20-f73b083ba6834771b07dbe5319917ae5) that they will delist monero on 2024-02-20, which means trading on the the above-mentioned pair will cease too.

besides, Binance has [routinely suspended](https://old.reddit.com/r/Monero/search?q=binance+withdraw&restrict_sr=on&sort=relevance&t=all) monero withdrawals in high-volatility scenarios, as was the [case](https://old.reddit.com/r/Monero/comments/1ak9lxf/all_cheap_xmr_has_been_bought_and_withdrawn_from/) with this event too. that makes it likely that Binance has been operating a fractional reserve for monero, rendering their prices unreliable even now.

I urge you to remove Binance from the monero Bisq Price Index constituents.

**Comment by pazza83** [comment]
I agree it should be removed. There is a risk prices become inaccurate if Binances XMR/BTC price diverges from other exchanges in the coming weeks

**Comment by dutu** [comment]
Poloniex also had XMR withdrawals suspended for many weeks, which makes the price to be constantly lower than other exchanges.
[Abnormal Exchange rate provider price impacting Bisq Market Rates](https://github.com/bisq-network/bisq-pricenode/issues/47)

On the other hand KuCoin (not used at the moment for price calculation) is having pretty good volumes on XMR/BTC (double than Kraken).

**Comment by ghost** [comment]
FYI these are the only exchanges currently feeding XMR price.  Removing Polo and Binance would leave just 2.  (The Polo price is automatically removed for deviating too much anyway).

Bitfinex: BTC/XMR: 0.0030139
Binance: BTC/XMR: 0.003025
Kraken: BTC/XMR: 0.003039 
Poloniex: BTC/XMR: 0.00295

**Reply by dutu** [reply]
Binance will have to be removed anyway since XMR will be delisted

**Reply by ghost** [reply]
Here is the PR to delist Binance/XMR: https://github.com/bisq-network/bisq-pricenode/pull/48

**Comment by ghost** [comment]
Feedback sought for the proposal to add a new **KuCoin** XMR price feed:
https://github.com/bisq-network/bisq-pricenode/issues/49
